fury/animation/tests/test_interpolators.py

```diff
@@ -0,0 +1,210 @@
+from itertools import combinations
+import numpy.testing as npt
+from fury.animation.interpolator import linear_interpolator, \
+    step_interpolator, cubic_spline_interpolator, cubic_bezier_interpolator, \
+    spline_interpolator, hsv_color_interpolator, lab_color_interpolator, \
+    xyz_color_interpolator, slerp
+from fury.testing import *
+
+
+def assert_not_equal(x, y):
+    npt.assert_equal(np.any(np.not_equal(x, y)), True)
+
+
+def test_step_interpolator():
```

Comment:
great ! you should do the same for test_helpers.py

also, check errors!  
- empty keyframes,  
- wrong key (not value)
- list instead of np.array, 
- etc...